add-on/src/lib/options.js

```diff
@@ -13,7 +13,7 @@ const optionDefaults = Object.freeze({
   customGatewayUrl: 'http://127.0.0.1:8080',
   ipfsApiUrl: 'http://127.0.0.1:5001',
   ipfsApiPollMs: 3000,
-  ipfsProxy: false
+  ipfsProxy: true
```

Comment:
I agree it is easier to keep it enabled during development, but we need to switch it off by default before this PR is merged :)